great_expectations/execution_engine/execution_engine.py

```diff
@@ -606,7 +606,7 @@ def _split_table_metric_domain_kwargs(
                 unexpected_keys_str: str = ", ".join(
                     map(lambda element: f'"{element}"', unexpected_keys)
                 )
-                logger.warning(
+                logger.info(
```

Comment:
@cdkini I believe that this is indeed a warning (as opposed to an informational message).  This situation must never happen -- and contributors included ourselves should be able to see this.  In other words, a user, who is not doing development in Great Expectations should never encounter this message.  Thank you.